source/System.cpp

```diff
@@ -599,6 +622,17 @@ bool System::Hidden() const
 
 
 
+// Return how much ramscoop is generated by this systems, depending the given ship ramscoop value.
+double System::RamscoopFuel(double shipRamscoop, double scale) const
+{
+	// Even if a ship has no ramscoop, it can harvest a tiny bit of fuel by flying close to the star,
+	// provided the system allows it.
+	double universal = 0.05 * scale * universalRamscoop;
+	return SolarWind() * .03 * scale * ramscoopMultiplier * (sqrt(shipRamscoop) + universal) + ramscoopAddend;
```

Comment:
Don't let a negative added steal fuel.
```suggestion
	return max(0., SolarWind() * .03 * scale * ramscoopMultiplier * (sqrt(shipRamscoop) + universal) + ramscoopAddend);
```